metroae

```diff
@@ -483,20 +489,30 @@ function destroy {
     fi
 
     set +e
-    docker rmi -f $IMAGE_ID
+    if [[ "$CONTAINER_STOPPED" -eq "1" ]] && [[ "$CONTAINER_STOPPED" -eq "1" ]]; then
```

Comment:
You are checking CONTAINER_STOPPED twice, did you mean CONTAINER_DELETED?

Reply:
Yes I did, it's fixed now, thanks!